test/models/building_test.dart

```diff
@@ -1,38 +1,25 @@
-import 'package:concordia_navigation/models/building.dart';
+import 'package:concordia_navigation/models/outdoor/building.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 
 void main() {
   group('Building', () {
-    test('constructor creates a list of edges', () {
-      List<LatLng> buildingEdges = [
+    List<LatLng> buildingEdges;
+    Building building;
+
+    setUp(() {
+      // Mock data
+      buildingEdges = [
         LatLng(45.49738, -73.57833),
         LatLng(45.49755, -73.57869),
         LatLng(45.49772, -73.57903),
       ];
-      Building building =
-          new Building(buildingName: 'name', buildingEdges: buildingEdges);
-      expect(building.buildingEdges.length, buildingEdges.length);
+      building = Building('TEST', 'Mock building', 1.0, 2.0, "123 mock address",
+          buildingEdges, "assets/mock_logo.png");
```

Comment:
I don't think this will work because Building will try to load that 'assets/mock_logo.png' which does not exist